### PR TITLE
Catd pipe

### PR DESCRIPTION
**Pull request by kingmushroom**

### CODE REVIEW

File: pipeline.yml [hunk 1]
```diff
@@ -1,1046 +1,1038 @@
...
-
-  - name: functions-requirements
-    type: pull-request
-    check_every: 24h
-    icon: github-circle
-    webhook_token: ((webhook_token-concourse))
+      access_token: ((github.concourse_access_key))
+######################################################################################
+#######                            Step funk                                     #####
+######################################################################################
```
Comment (Jordancooke) [inline pipeline.yml]: I noticed there is both a "step funk" and "Step function section". Any reason for this replication? If not could they be combined ?

File: pipeline.yml [hunk 2]
```diff
@@ -1,1046 +1,1038 @@
...
+      aws_role_arn: REDACTED
+      aws_access_key_id: ((aws.service_account_access_key_secret))
+      aws_secret_access_key: ((aws.service_account_access_key_id))
+      aws_region: eu-west-2
+
+  - name: pipeline-pull-repository
     type: pull-request
-    check_every: 24h
+    check_every: 2m
     webhook_token: ((webhook_token-concourse))
```
Comment (Jordancooke) [inline pipeline.yml]: With the removal of webhook's and the implementation of Polling is the webhook token still required? If not could it be removed?

File: pipeline.yml [hunk 3]
```diff
@@ -1,1046 +1,1038 @@
...
-      - enrichment-deploy
-      - strata-deploy
-      - imputation-deploy
-      - aggregation-deploy
-      - disclosure-deploy
-      - success-handler-deploy
-      - error-capture-deploy
-      - step-function-deploy
-
-
```
Comment (Jordancooke) [inline pipeline.yml]: Any reason for the removal of the display groups?

File: pipeline.yml [hunk 4]
```diff
@@ -1,1046 +1,1038 @@
...
+            path: sh
+            args:
+              - -ec
+              - |
+                eval $CREDS
+                aws secretsmanager get-secret-value --secret-id results-bmi-wranglers \
+                | jq -r '.SecretString' >> wranglers.json
+                es-bmi-step-functions-deploy-repository/serverless.sh
+
+  ########################################## #############################################
```
Comment (Jordancooke) [inline pipeline.yml]: I think this chain is broken ⛓

File: pipeline.yml (hunk 3 above)
Comment (kingmushroom) [reply pipeline.yml]: No seperate installing dependencies step now, so i felt the pipe wasnt big enough to warrant it. I'd be happy to add in though if thats whats wanted

File: pipeline.yml (hunk 2 above)
Comment (kingmushroom) [reply pipeline.yml]: good point

File: pipeline.yml (hunk 1 above)
Comment (kingmushroom) [reply pipeline.yml]: Good spot, step funk was a placeholder that i forgot to get rid of

**Pull request merged**